Add support for Wacom Movink

flying-elephant opened the pull request:
Those commits are ready for Wacom Movink announced yesterday.

flying-elephant commented on data/movink.tablet:
```diff
@@ -0,0 +1,42 @@
+Class=Movink
+Width=11
+Height=8
+Styli=@udpen;
```
>This tablet is not compatible with current AES styli or any other previous styli? 
No, not at all. Movink doesn't work with AES styli because the used technology is much different from it.
Although the technology in udpen is just like that of ProPen or our other branded pens, it cannot work with our previous products. Therefore, adding "udpen' into "Styli" line is needed for Movink.

flying-elephant commented on data/movink.tablet:
```diff
@@ -0,0 +1,42 @@
+Buttons=3
+
+[Buttons]
+Left=A;C
+Right=B;C
```
>Top=C, right?
I'll work on this, thanks!
There are two buttons on each sides, so I thought they should be the sides.
"Tpo=C" makes much sense.

Yes, one of the physical buttons can be assigned for an express key, but it also can be other functionalities than the express key; one of them is "Touch on/off".

whot replied:
> can be assigned for an express key, 

so these are software/firmware controlled buttons? that's going to be ... interesting, libwacom doesn't have a good way to express these for now.

Might be best to add a comment in the file here and then figure out how to really handle this later when we have the consumers (e.g. mutter + gnome-control-center) actually wanting deal with these buttons.

whot commented on data/libwacom.stylus:
```diff
@@ -749,3 +749,22 @@ Type=Puck
+PairedStylusIds=0x10002;
+EraserType=Invert;
+Axes=Tilt;Pressure;Distance;
+Type=General
```
Missing a `Buttons` line. Looks like the old pens all had 1 button for the eraser but I  no longer remember why this was the case... I think this predates the distinction between tip and buttons so `Buttons=1` is necessary here.

whot commented on data/movink.tablet:
```diff
@@ -0,0 +1,42 @@
+# Movink 13HD touch
+# DTH-135
+#
+# Note: touch data comes through the usb:056a:03f0 interface
```
this is missing the sysinfo line (see the bot comment)

whot reviewed (changes requested):
Thanks for the update. CI still fails so at least two changes are still needed - please make sure to run `meson test -C builddir` locally, that'll run (most of) the test suite locally so it'll speed up finding the various nitpicks my human eyes won't see :)

flying-elephant commented:
I modified the codes and updated them.
However, I got the attached log by running the suggestion, "meson test -C".
Do you know what is wrong exactly?
The log looks like the lines which describe about the eraser button, but I'm not sure what I should do.
"UD pen" could have an eraser button, so it has to stick with it somehow.
[testlog.txt](https://github.com/linuxwacom/libwacom/files/15175576/testlog.txt)

whot commented on data/libwacom.stylus:
```diff
@@ -749,3 +749,23 @@ Type=Puck
+Group=udpen
+PairedStylusIds=0x10002;
+Buttons=1
+EraserType=Invert;
```
```suggestion
EraserType=Invert
```

whot commented on data/libwacom.stylus:
```diff
@@ -749,3 +749,23 @@ Type=Puck
+Group=udpen
+PairedStylusIds=0x1000a;
+Buttons=2
+EraserType=Invert;
```
```suggestion
EraserType=Invert
```
This is a single value item so it probably fails to parse this one

whot reviewed (commented):
I think the eraser parsing fails, fixing that should fix the CI

whot commented on data/movink.tablet:
```diff
@@ -0,0 +1,45 @@
+Name=Wacom Movink
+ModelName=DTH-135K0C
+DeviceMatch=usb|056a|03f0
+Class=Movink
```
```suggestion
Class=Cintiq
```

`Class` is deprecated, no-one uses it and this is close enough to a cintiq so let's go with that.

whot commented on data/libwacom.stylus:
```diff
@@ -749,3 +749,23 @@ Type=Puck
+Group=udpen
+PairedStylusIds=0x1000a;
+Buttons=2
+EraserType=Invert
```
```suggestion
```

EraserType needs to be set on the eraser, not on the pen

whot commented on data/movink.tablet:
```diff
@@ -0,0 +1,45 @@
+#    |                                          |
+#    *------------------------------------------*
+#
+# Note: touch data comes through the usb:056a:03f0 interface
+#
```
```suggestion
# Note: This tablet has 2 physical buttons (one left, one right)
# configured via a firmware OSD but only one button can send 
# events at any time. This is button C, shown centered here.
# We cannot tell which physical button triggered the event.
#
# Note: touch data comes through the usb:056a:03f0 interface
#
```

whot commented on data/movink.tablet:
```diff
@@ -0,0 +1,45 @@
+Buttons=3
+
+[Buttons]
+Left=A;C
```
```suggestion
# Button C can be physically left or right depending on the firmware
# but let's assign it to Left here, that's good enough.
Left=A;C
```

whot commented:
All good now, let's get this merged! Thanks heaps for the effort
